assets/scripts/app/BlockingError.jsx

```diff
@@ -254,6 +262,24 @@ function BlockingError (props) {
         </>
       )
       break
+    case ERRORS.AUTH_FAILURE:
+      title = (
+        <FormattedMessage
+          id="error.auth-failure"
+          defaultMessage="Your authentication has expired."
```

Comment:
1. Rename the message ID to `error.auth-expired`
2. The default message should be "Your session has expired."
3. Add this message to `./assets/locales/en/main.json`.